src/components/atoms/IconInput/IconInput.scss

```diff
@@ -0,0 +1,41 @@
+@import "scss/constants";
+
+.icon-input {
+  position: relative;
+
+  &__input {
+    height: 100%;
+
+    &--icon-from-left {
+      padding-left: 2.5rem;
+    }
+
+    &--icon-from-right {
```

Comment:
```suggestion
    &--icon-right {
```

Reply:
It might be better to move these variants onto the base class itself, so that they are:

- `.icon-input--icon-left`
- `.icon-input--icon-right`

or similar.

Reply:
renamed the classNames, As we decided to use `iconStart` and `iconEnd` prop names, I assumed that it'd be more logical to give class names similar names:
* `input-field--icon-start`
* `input-field--icon-end`

commit: https://github.com/sparkletown/sparkle/pull/1306/commits/4522f27ae790a9bc30e71c2e4e06433f7e2b9141